stories/components/picker/picker.html.stories.js

```diff
@@ -25,6 +25,11 @@ export const MaxPages = () => html`
 export const SelectedValue = () => html`
   <mgt-picker resource="/groups" selected-value="Activewear" scopes="group.read.all" key-name="displayName"></mgt-picker>
 `;
+
+export const NestedValues = () => html`
+  <mgt-picker resource="/applications?$count=true" placeholder="Select application home pages" scopes="application.read.all, directory.read.all" key-name="web.homePageUrl" cache-enabled="true" cache-invalidation-period="50000"></mgt-picker>
```

Comment:
Thanks for that! As this endpoint isn't great in the sandbox tenant, I'd offer the following suggestion:

```suggestion
  <mgt-picker resource="teams/02bd9fd6-8f93-4758-87c3-1fb73740a315/installedApps?$expand=teamsAppDefinition" placeholder="Select teams applications" scopes="teamsappinstallation.readforteam" key-name="teamsAppDefinition.displayName" cache-enabled="true" cache-invalidation-period="50000"></mgt-picker>
```

If not, it just looks... weird!

![image](https://github.com/microsoftgraph/microsoft-graph-toolkit/assets/7620955/8f065145-0e6f-457b-a492-937087c3befe)

Reply:
Thanks for the suggestion.